### ISSUE TITLE
[Severe] Controls and media elements should have labels

### ISSUE THREAD

**Issue by rodowi** [mxabierto/participa#228]
[Severe] Controls and media elements should have labels (5)
https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_01
```html
<input id="doc-text-filter" type="text" ng-model="docSearch" class="form-control ng-pristine ng-valid" placeholder="Título">
<input class="select2-focusser select2-offscreen" type="text" id="s2id_autogen1">
<input class="select2-focusser select2-offscreen" type="text" id="s2id_autogen2">
<input class="form-control ng-pristine ng-valid ng-valid-email" id="email" placeholder="Ingresa tu correo electrónico" type="email" ng-model="user">
<button class="blue-right btn" id="subscribe" type="button" ng-click="subscribeEmail()">

**Comment by rodowi** [comment]
Encontrado en:
- /consulta
-  /consulta/auth/login
